tests/telemetry/test_writer.py

```diff
@@ -579,6 +579,12 @@ def test_app_started_event_configuration_override(test_agent_session, run_python
         {"name": "_DD_APPSEC_DEDUPLICATION_ENABLED", "origin": "default", "value": True},
         {"name": "_DD_IAST_LAZY_TAINT", "origin": "default", "value": False},
         {"name": "_DD_IAST_USE_ROOT_SPAN", "origin": "default", "value": False},
+        {"name": "_DD_NATIVE_LOGGING_BACKEND", "origin": "default", "value": "file"},
```

Comment:
Do we want to submit these configs to the instrumentation telemetry platform? If so, we should also update the normalization rules: https://github.com/DataDog/dd-go/blob/prod/trace/apps/tracer-telemetry-intake/telemetry-payload/static/config_norm_rules.json

Reply:
I added them to the python specific ones: https://github.com/DataDog/dd-go/blob/d374ad0fdcd0c60e1653e142ebd31fdf2cd2eea1/trace/apps/tracer-telemetry-intake/telemetry-payload/static/python_config_rules.json#L5